build/docker/Dockerfile

```diff
@@ -20,7 +22,7 @@ LABEL description="Honeydipper - an event-driven orchestration framework" \
 
 RUN apk add --no-cache ca-certificates git
 
-WORKDIR /opt/honeydipper/drivers/builtin
-COPY --from=build /go/bin/* ./
+COPY --from=build /build/honeydipper ./app/honeydipper
+
```

Comment:
We should also set a non-root user in here, but that wasn't setup before either. We can handle that separately in another PR.

After the integration test is passing, we'll probably still do a test on our end to make sure the Docker image still builds / runs Ok.

Reply:
Sounds good, will followup with a PR to use non-root and verify separately if that's OK?

Reply:
Yes, or we can even do it. So no need to change that in this PR.

Btw, we have a Gitter channel; would love to hear if you’re using Honeydipper and / or what brought you to the project.